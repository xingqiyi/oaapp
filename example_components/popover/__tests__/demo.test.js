import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('popover');
