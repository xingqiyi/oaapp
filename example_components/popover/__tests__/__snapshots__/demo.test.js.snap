// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/popover/demo/basic.md correctly 1`] = `
<div>
  <div
    class="am-navbar am-navbar-light"
  >
    <div
      class="am-navbar-left"
      role="button"
    >
      <span
        class="am-navbar-left-content"
      />
    </div>
    <div
      class="am-navbar-title"
    >
      NavBar
    </div>
    <div
      class="am-navbar-right"
    >
      <div
        style="height:100%;padding:0 0.3rem;margin-right:-0.3rem;display:flex;align-items:center;"
      >
        <svg
          class="am-icon am-icon-ellipsis am-icon-md"
        >
          <use
            xlink:href="#ellipsis"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
